packages/streaming-image-volume-loader/src/helpers/splitImageIdsBy4DTags.ts

```diff
@@ -0,0 +1,68 @@
+import { metaData } from '@cornerstonejs/core';
+
+// Supported 4D Tags
+//   (0018,1060) Trigger Time                   [NOK]
+//   (0018,0081) Echo Time                      [NOK]
+//   (0018,0086) Echo Number                    [NOK]
+//   (0020,0100) Temporal Position Identifier   [NOK]
+//   (0054,1300) FrameReferenceTime             [OK]
+
+interface MappedFrameReferenceTime {
+  imageId: string;
+  frameReferenceTime: number;
+}
+
+const groupBy = (array, key) => {
+  return array.reduce((rv, x) => {
+    (rv[x[key]] = rv[x[key]] || []).push(x);
+    return rv;
+  }, {});
+};
+
+function splitFramesByFrameReferenceTime(imageIds: string[]): string[][] {
+  const framesMetadata: Array<MappedFrameReferenceTime> = imageIds.map(
+    (imageId: string): MappedFrameReferenceTime => {
+      const petImageModule = metaData.get('petImageModule', imageId);
```

Comment:
should we add a todo for other tags other than pt? e.g., mr, fmri? etc.?

Reply:
Added a  "TODO" at the top of this file.